cypress/page-objects/lib/ui-elements.ts

```diff
@@ -0,0 +1,37 @@
+export class UIElement {
```

Comment:
I didn't notice that before, but why it's elementS in plural in the filename?

Reply:
my bad